GrandCentralBoard/Core/Configuration/WidgetSettings.swift

```diff
@@ -22,6 +22,36 @@ public struct WidgetSettings {
     }
 }
 
+extension WidgetSettings : Hashable {
+    public var hashValue: Int {
+
+        if let settings = settings as? [String : AnyObject] {
+            return settings.enumerate().reduce(name.hashValue) { first, second in
```

Comment:
`first` and  `second` should be named according to their true meaning, such as `result` and `hash`.